package.json

```diff
@@ -96,6 +97,98 @@
             "type": "string"
           },
           "markdownDescription": "Optional list of custom resolvers passed to Coursier when fetching metals dependencies.\n\nFor documentation on accepted values see the [Coursier documentation](https://get-coursier.io/docs/other-repositories).\n\nThe extension will pass these to Coursier using the COURSIER_REPOSITORIES environment variable after joining the custom repositories with a pipe character (|)."
+        },
+        "treeviews.initialWidth": {
+          "type": "number",
+          "default": 40,
+          "description": "Initial TreeViews panel width"
+        },
+        "treeviews.alignment": {
+          "type": "string",
+          "default": "left",
+          "enum": [
+            "left",
+            "right"
+          ],
+          "description": "TreeViews panel position"
+        },
+        "treeviews.initialViews": {
+          "type": "object",
+          "default": [
+            {
+              "name": "metalsCompile",
+              "size": 10,
+              "expanded": [
+                [
+                  "metals://ongoing-compilations"
+                ]
+              ]
+            },
+            {
+              "name": "metalsBuild",
+              "size": 40,
+              "expanded": [
+                [
+                  "projects:"
+                ]
+              ]
+            }
+          ]
+        },
+        "treeviews.toggleNode": {
```

Comment:
Are we able to prepend metals to all of these to stay consistent with the others. For example `metals.treeviews.toggleNode`.

Reply:
Sure. Will do soon.

Reply:
Done.